### PR TITLE
Bug 1304489, edited AccessModes content

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1304489

Original PR: https://github.com/openshift/openshift-docs/pull/1665

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tpoitras @bfallonf can you please eyeball the mark-up here? Thanks!

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -219,13 +221,14 @@ The access modes are:
...
 ====
-It is important to note that a volume's AccessModes are descriptors of the volume's capabilities, they are not enforced constraints.
-The storage provider is responsible for runtime errors resulting from invalid use of the resource.
-
-For example, a GCE Persistent Disk has AccessModes *ReadWriteOnce* and *ReadOnlyMany*.  It is left to the user to mark their
-claims as read only if they wish to take advantage of the volume's ability for ROX.  Errors in the provider show up at runtime
-as mount errors.
-
+It is important to note that a volume's `*AccessModes*` are descriptors of the
+volume's capabilities, they are not enforced constraints. The storage provider
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Comma splice above... Is there a word missing from the sentence? Maybe:

"A volume's Accesmodes are descriptors of the volume's capabilities, but they are not enforced constraints."?

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -219,13 +221,14 @@ The access modes are:
...
+is responsible for runtime errors resulting from invalid use of the resource.
+
+For example, a GCE Persistent Disk has `*AccessModes*` *ReadWriteOnce* and
+*ReadOnlyMany*. It is left to the user to mark their claims as `read-only` if they
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Some commas inbetween the three modes above?

File: architecture/additional_concepts/storage.adoc [hunk 3]
```diff
@@ -192,9 +192,11 @@ For example, NFS can support multiple read/write clients, but a specific NFS PV
 might be exported on the server as read-only. Each PV gets its own set of access
 modes describing that specific PV's capabilities.
 
-Claims are matched to Volumes with similar access modes.  Because a claim's access modes represent a request,
-the user may be granted more but never less.  For example, if a claim requests RWO but the only volume available was
-  an NFS PV (RWO+ROX+RWX), the claim would match NFS because it supports RWO.  Direct matches are always attempted first.
+Claims are matched to Volumes with similar access modes. A claim's access modes
+represent a request. Therefore, the user may be granted more, but never less.
+For example, if a claim requests RWO but the only volume available was an NFS PV
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Comma after "RWO"?

**Comment by bfallonf** [comment]
@ahardin-rh I made a few suggestions. I realise they were all pretty much there when you got to this file, but thought I'd bring them up while you're in there messing around. Otherwise, looks good to me :fireworks:

File: architecture/additional_concepts/storage.adoc [hunk 4]
```diff
@@ -219,13 +221,14 @@ The access modes are:
...
-claims as read only if they wish to take advantage of the volume's ability for ROX.  Errors in the provider show up at runtime
-as mount errors.
-
+It is important to note that a volume's `*AccessModes*` are descriptors of the
+volume's capabilities, they are not enforced constraints. The storage provider
+is responsible for runtime errors resulting from invalid use of the resource.
+
+For example, a GCE Persistent Disk has `*AccessModes*` *ReadWriteOnce* and
+*ReadOnlyMany*. It is left to the user to mark their claims as `read-only` if they
+wish to take advantage of the volume's ability for ROX. Errors in the provider
```
Comment (tpoitras) [inline architecture/additional_concepts/storage.adoc]: s/wish/want

As per IBM Style Guide: "wish": Do not use. Use "want".

**Comment by ahardin-rh** [comment]
**revhistory note**

    (for link:../architecture/additional_concepts/storage.html[Persistent Storage]

Updated the link:../architecture/additional_concepts/storage.html#pv-access-modes[Access Modes] content to address how access mode matching works and whether modes are enforced at runtime.

**Pull request merged**